bot/extensions/activities.py

```diff
@@ -0,0 +1,38 @@
+from discord.ext.commands import Cog, command
+import os
+import requests
+import json
+import discord
+
+
+class Activities(Cog, name="Activities",description="Commands regarding Discord Activities in VC" ):
```

Comment:
The name is not really necessary here since the name of the class is already **Activities**, although it doesn't change much. Also, don't forget to insert spaces after your attributes. In this case, between the **name** and **description**.